### DISCUSSION TITLE
make all 'elemts' skinnable [like in stable]

### DISCUSSION THREAD

**Discussion by murryguy** [ppy/osu#28328] [Ideas]
i wana make my song 'seletet' skinnable
i wana make more ingame elements skinnable
pls add this

**Comment by murryguy** [comment]
PLEASE ANSWER

**Comment by BlockHead007** [comment]
Don't create new discussions if there is already existing topic, especially if you know it exists. 
#28266, #27353 and maybe some others

**Reply by murryguy** [reply]
> Don't create new discussions if there is already existing topic, especially if you know it exists. #28266, #27353 and maybe some others

oh..
